terraform/cp4aiops/main.tf

```diff
@@ -1,111 +1,88 @@
+# Getting the OpenShift cluster configuration
+data "ibm_resource_group" "group" {
+  name = var.resource_group
 }
 
-module "cluster" {
-  // source = "../../../../ibm-hcbt/terraform-ibm-cloud-pak/roks"
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
-  enable = local.enable_cluster
-  on_vpc = var.on_vpc
-
-  // General parameters:
-  project_name = var.project_name
-  owner        = var.owner
-  environment  = var.environment
+resource "null_resource" "mkdir_kubeconfig_dir" {
+  triggers = { always_run = timestamp() }
+  provisioner "local-exec" {
+    command = "mkdir -p ${var.cluster_config_path}"
+  }
+}
 
-  // Openshift parameters:
+module "create_cluster" {
+//  source = "../../../terraform-ibm-cloud-pak/modules/roks" # terraform-ibm-cloud-pak/modules/roks
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/roks"
+  enable               = local.enable_cluster
+  on_vpc               = var.on_vpc
+  project_name         = var.project_name
+  environment          = var.environment
+  owner                = var.owner
   resource_group       = var.resource_group
   roks_version         = var.roks_version
   flavors              = var.flavors
   workers_count        = var.workers_count
   datacenter           = var.datacenter
   force_delete_storage = true
-  vpc_zone_names       = var.vpc_zone_names
-
-  // Kubernetes Config parameters:
-  // download_config = false
-  // config_dir      = local.kubeconfig_dir
-  // config_admin    = false
-  // config_network  = false
-
-  // Debugging
-  private_vlan_number = var.private_vlan_number
-  public_vlan_number  = var.public_vlan_number
-}
-
-resource "null_resource" "mkdir_kubeconfig_dir" {
-  triggers = { always_run = timestamp() }
-
-  provisioner "local-exec" {
-    command = "mkdir -p ${var.config_dir}"
-  }
+  private_vlan_number  = var.private_vlan_number
+  public_vlan_number   = var.public_vlan_number
 }
 
 data "ibm_container_cluster_config" "cluster_config" {
   depends_on = [null_resource.mkdir_kubeconfig_dir]
-
-  cluster_name_id   = local.enable_cluster ? module.cluster.id : var.cluster_id
-  resource_group_id = module.cluster.resource_group.id
-  config_dir        = var.config_dir
+  cluster_name_id   = local.enable_cluster ? module.create_cluster.id : var.cluster_id
+  resource_group_id = module.create_cluster.resource_group.id
+  config_dir        = var.cluster_config_path
   download          = true
   admin             = false
   network           = false
 }
 
-module "portworx" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/portworx"
-  // TODO: With Terraform 0.13 replace the parameter 'enable' or the conditional expression using 'with_iaf' with 'count'
+module "install_portworx" {
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/portworx"
```

Comment:
This will need to be updated. The terraform-0.13 branch has been merged to main.

Update to github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/portworx